charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -91,6 +91,27 @@ spec:
       {{- end }}
       volumeSnapshotLocation: []
     metrics:
+      prometheusRule:
+        additionalLabels: {{- .Values.monitoring.labels | toYaml | nindent 10 }}
+        enabled: true
```

Comment:
This should also only be enabled if prometheus is activated, just like with the ServiceMonitor, so I would just not set this, as this is also autodetected